autospec/config.py

```diff
@@ -265,6 +266,7 @@
     (r"ImportError:..*: No module named ([a-zA-Z0-9\-\._]+)", 0, 'pypi'),
     (r"ImportError: No module named ([a-zA-Z0-9\-\._]+)", 0, 'pypi'),
     (r"ImportError: No module named '([a-zA-Z0-9\-\._]+)'", 0, 'pypi'),
+    (r"ImportError: No module named '([a-zA-Z0-9\-\._]+)'", 0, 'pypi'),
```

Comment:
I don't see a difference between this pattern and the previous one... Is this intentional?

Reply:
Nope, that's a mistake, thanks